src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -1405,10 +1405,8 @@ def onPolyModelChange(self, top, bottom):
                 if parameter_name_w in self.poly_params_to_fit:
                     self.poly_params_to_fit.remove(parameter_name_w)
             self.cmdFit.setEnabled(self.haveParamsToFit())
-            # force data update
-            key = parameter_name + '.' + delegate.columnDict()[delegate.poly_pd]
-            self.poly_params[key] = value
-            self.updateData()
```

Comment:
The point of the above assignment was to redo the model/chart update on **selecting** a polydisperse parameter.
SasView 5 is trying to be "interactive", meaning chart/model updates happen on any parameter change.
So, if you enter a PD value and switch focus from the cell, the chart updates properly. 
However I can see that this does not correspond to what we do with main parameters. As such, this is probably a better solution in terms of functionality.